diff --git a/node_modules/react-native-pdf/fabric/RNPDFPdfNativeComponent.js b/node_modules/react-native-pdf/fabric/RNPDFPdfNativeComponent.js
index 596d796..7122833 100644
--- a/node_modules/react-native-pdf/fabric/RNPDFPdfNativeComponent.js
+++ b/node_modules/react-native-pdf/fabric/RNPDFPdfNativeComponent.js
@@ -4,8 +4,8 @@
  */
  'use strict';
 
- import codegenNativeComponent from 'react-native/Libraries/Utilities/codegenNativeComponent';
  import codegenNativeCommands from 'react-native/Libraries/Utilities/codegenNativeCommands';
+import codegenNativeComponent from 'react-native/Libraries/Utilities/codegenNativeComponent';
 
  type ChangeEvent = $ReadOnly<{|
    message: ?string,
@@ -22,6 +22,7 @@
    enablePaging: ?boolean,
    enableRTL: ?boolean,
    enableAnnotationRendering: ?boolean,
+   enableDoubleTapZoom: ?boolean,
    showsHorizontalScrollIndicator: ?boolean,
    showsVerticalScrollIndicator: ?boolean,
    enableAntialiasing: ?boolean,
